.github/workflows/ci.yaml

```diff
@@ -99,6 +103,7 @@ jobs:
             additionalTags:
               - materialize-timescaledb
               - materialize-alloydb
+              - materialize-amazonaurora
```

Comment:
for the sake of convention / predictability of names, let's also make this `materialize-amazon-aurora`

Reply:
should I add -postgres too just to be consistent with how we named in for the source?